Src/Lib/DownloadHelper.cpp

```diff
@@ -848,6 +745,85 @@ void VersionData::Swap( VersionData &data )
 	std::swap(languages,data.languages);
 }
 
+// very simple json "parser"
+// it can parse simple json values in form:
```

Comment:
Impressive, but I would rather use a library in the future for added robustness. There are many minimal, extremely fast libraries available.